indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -204,6 +204,9 @@ public TaskStatus call()
                               command.add(String.format("-Ddruid.host=%s", childHost));
                               command.add(String.format("-Ddruid.port=%d", childPort));
 
+                              command.add("-XX:+UseThreadPriorities");
+                              command.add("-XX:ThreadPriorityPolicy=42");
```

Comment:
https://docs.oracle.com/cd/E15289_01/doc.40/e15062/optionxx.htm#BABGBFHF

Without setting the command line flags the behavior is disabled at the JVM level